### PR TITLE
Blog Authentication

### PR DESCRIPTION
**Pull request by cabilangan112**

### CODE REVIEW

File: jassen/project/blog/models.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    heading = models.CharField(max_length=150)
+    sub_heading = models.CharField(max_length=150)
+    
+    def __str__(self):
+        return '{}'.format(self.heading)
+
+class Post(models.Model):
+    user = models.ForeignKey(User,on_delete=models.CASCADE)
+    title = models.CharField(max_length=150)
+    sub_Title = models.CharField(max_length=150)
```
Comment (rocity) [inline jassen/project/blog/models.py]: @cabilangan112 model field names must be in lower case

File: jassen/project/blog/models.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    banner_photo = models.ImageField(upload_to = 'static/media')
+    body = models.TextField()
+    date = models.DateTimeField(auto_now_add=True)
+    date_modified = models.DateTimeField(auto_now_add=True)
+    blog = models.ForeignKey(Blog, on_delete=models.CASCADE) 
+    category = models.ForeignKey('Category', on_delete=models.CASCADE)
+    tags = models.ManyToManyField("Tag",related_name="tags")
+    status = models.CharField(max_length=9, choices=POST_STATUS, blank=True, default=True)
+    Comment = models.ManyToManyField("comment",related_name="Comment")
```
Comment (rocity) [inline jassen/project/blog/models.py]: @cabilangan112 model field names must be in lower case

File: jassen/project/blog/models.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    
+    def __str__(self):
+        return '{}'.format(self.user)
+
+class Comment(models.Model):
+    user = models.ForeignKey(User,on_delete=models.CASCADE)
+    vote = models.BooleanField(default=True)
```
Comment (rocity) [inline jassen/project/blog/models.py]: @cabilangan112 do we need the `vote` field?

File: jassen/project/user/models.py [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+            password=password,
+            date_of_birth=date_of_birth,
+
+        )
+        user.is_admin = True
+        user.save(using=self._db)
+        return user
+
+
+class MyUser(AbstractBaseUser):
```
Comment (rocity) [inline jassen/project/user/models.py]: @cabilangan112 Don't name anything with `My`. Don't treat projects as test projects.

File: jassen/project/user/models.py [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    email = models.EmailField(
+        verbose_name='email address',
+        max_length=255,
+        unique=True,
+    )
+    cover_photo     = models.ImageField(upload_to = 'static/media')
+    profile_picture =  models.ImageField(upload_to = 'static/media')
+    last_name       =  models.CharField(max_length=100)
+    first_name      =  models.CharField(max_length=100)
+    MI              =   models.CharField(max_length=200)
```
Comment (rocity) [inline jassen/project/user/models.py]: @cabilangan112 what's this?

File: jassen/project/user/models.py [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+    Sex             =   models.CharField(max_length=6, choices=GENDER, blank=True, default=True)
```
Comment (rocity) [inline jassen/project/user/models.py]: @cabilangan112 model field names must be in lower case

File: jassen/project/user/models.py [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+    date_of_birth   =   models.DateField()
+    Age             =   models.CharField(max_length=20)
```
Comment (rocity) [inline jassen/project/user/models.py]: @cabilangan112 model field names must be in lower case

File: jassen/project/user/views.py [hunk 8]
```diff
@@ -0,0 +1,15 @@
+from django.shortcuts import render
+from django.contrib.auth import get_user_model
+from django.shortcuts import render
+from django.shortcuts import render, get_object_or_404, redirect
+from .forms import RegisterForm
+from django.views.generic import (ListView,DetailView,UpdateView,CreateView)
+# Create your views here.
+
+class RegisterView(CreateView):
+	form_class = RegisterForm
```
Comment (rocity) [inline jassen/project/user/views.py]: @cabilangan112 use spaces instead of tabs

File: jassen/project/user/forms.py [hunk 9]
```diff
@@ -0,0 +1,71 @@
+from django import forms
```
Comment (rocity) [inline jassen/project/user/forms.py]: @cabilangan112 use spaces instead of tabs

File: jassen/project/templates/registration/register.html [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+	                <div class="col-md-6 mx-auto">
+	                    <span class="anchor" id="formLogin"></span>
+
+	                    <!-- form card login -->
+	                    <div class="cards card rounded-1">
+	                        <div class="card-header">
+	                            <h3 class="mb-0">Sign-up</h3>
+	                        </div>
+	                        <div class="card-body">
+	  						<form method="post" enctype="multipart/form-data">{% csrf_token %}
```
Comment (rocity) [inline jassen/project/templates/registration/register.html]: @cabilangan112 indent this properly

File: jassen/project/user/views.py (hunk 8 above)
Comment (cabilangan112) [reply jassen/project/user/views.py]: unya nako mag pr ulit sir gna fix pa

File: jassen/project/user/forms.py (hunk 9 above)
Comment (rocity) [reply jassen/project/user/forms.py]: @cabilangan112 this file still uses tabs instead of spaces

**Pull request merged**